src/actions/auth.js

```diff
@@ -1,24 +1,14 @@
 import firebase from "../firebase";
 import { handleError } from "../utils";
 import { fetchProfile } from "./profile";
+import { asyncAction, editAction } from "commons/utils/action-creators";
 
-export const USER_NOT_LOGGED = "USER_NOT_LOGGED";
+export const USER_SIGN_IN = asyncAction("USER_SIGN_IN");
+export const USER_NOT_SIGN_IN = "USER_NOT_LOGGED";
```

Comment:
String needs updating :)

Reply:
sorry, working in multiple projects at the same time does that XD